### DISCUSSION TITLE
Educational establishments

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#78] [Planning consideration]
## Educational establishments

**Comment by colmjude** [comment]
### Existing data

It is possible to download establishment data using the [Get Information about Schools](https://get-information-schools.service.gov.uk/) service.

You have to choose what type of download you want so I'm not sure what the endpoint would be to collect this data.

Only the full download (All establishment data > Establishment fields CSV, 62.9 MB) has easting and northing fields that can be used to establish a point.

**Reply by colmjude** [reply]
An establishment record contains these fields

| Field | Value |
|-------|-------|
| URN | 100001 |
| LA (code) | 201 |
| LA (name) | City of London |
| EstablishmentNumber | 6005 |
| EstablishmentName | City of London School for Girls |
| TypeOfEstablishment (code) | 11 |
| TypeOfEstablishment (name) | Other independent school |
| EstablishmentTypeGroup (code) | 3 |
| EstablishmentTypeGroup (name) | Independent schools |
| EstablishmentStatus (code) | 1 |
| EstablishmentStatus (name) | Open |
| ReasonEstablishmentOpened (code) | 00 |
| ReasonEstablishmentOpened (name) | Not applicable |
| OpenDate | 01-01-1920 |
| ReasonEstablishmentClosed (code) | 00 |
| ReasonEstablishmentClosed (name) | Not applicable |
| CloseDate |  |
| PhaseOfEducation (code) | 0 |
| PhaseOfEducation (name) | Not applicable |
| StatutoryLowAge | 7 |
| StatutoryHighAge | 18 |
| Boarders (code) | 1 |
| Boarders (name) | No boarders |
| NurseryProvision (name) | No Nursery Classes |
| OfficialSixthForm (code) | 1 |
| OfficialSixthForm (name) | Has a sixth form |
| Gender (code) | 2 |
| Gender (name) | Girls |
| ReligiousCharacter (code) | 06 |
| ReligiousCharacter (name) | None |
| ReligiousEthos (name) | Inter- / non- denominational |
| Diocese (code) | 0000 |
| Diocese (name) | Not applicable |
| AdmissionsPolicy (code) | 2 |
| AdmissionsPolicy (name) | Selective |
| SchoolCapacity | 860 |
| SpecialClasses (code) | 2 |
| SpecialClasses (name) | No Special Classes |
| CensusDate | 20-01-2022 |
| NumberOfPupils | 739 |
| NumberOfBoys | 0 |
| NumberOfGirls | 739 |
| PercentageFSM | 0.0 |
| TrustSchoolFlag (code) | 0 |
| TrustSchoolFlag (name) | Not applicable |
| Trusts (code) |  |
| Trusts (name) |  |
| SchoolSponsorFlag (name) | Not applicable |
| SchoolSponsors (name) |  |
| FederationFlag (name) | Not applicable |
| Federations (code) |  |
| Federations (name) |  |
| UKPRN | 10013279 |
| FEHEIdentifier |  |
| FurtherEducationType (name) | Not applicable |
| OfstedLastInsp |  |
| OfstedSpecialMeasures (code) | 0 |
| OfstedSpecialMeasures (name) | Not applicable |
| LastChangedDate | 10-06-2024 |
| Street | St Giles' Terrace |
| Locality | Barbican |
| Address3 |  |
| Town | London |
| County (name) |  |
| Postcode | EC2Y 8BB |
| SchoolWebsite | http://www.clsg.org.uk |
| TelephoneNum | 02078475500 |
| HeadTitle (name) | Mrs |
| HeadFirstName | Jenny |
| HeadLastName | Brown |
| HeadPreferredJobTitle | Headteacher |
| BSOInspectorateName (name) | Not applicable |
| InspectorateReport |  |
| DateOfLastInspectionVisit |  |
| NextInspectionVisit |  |
| TeenMoth (name) | Not applicable |
| TeenMothPlaces |  |
| CCF (name) | Not applicable |
| SENPRU (name) | Not applicable |
| EBD (name) | Not applicable |
| PlacesPRU |  |
| FTProv (name) |  |
| EdByOther (name) | Not applicable |
| Section41Approved (name) | Not approved |
| SEN1 (name) |  |
| SEN2 (name) |  |
| SEN3 (name) |  |
| SEN4 (name) |  |
| SEN5 (name) |  |
| SEN6 (name) |  |
| SEN7 (name) |  |
| SEN8 (name) |  |
| SEN9 (name) |  |
| SEN10 (name) |  |
| SEN11 (name) |  |
| SEN12 (name) |  |
| SEN13 (name) |  |
| TypeOfResourcedProvision (name) |  |
| ResourcedProvisionOnRoll |  |
| ResourcedProvisionCapacity |  |
| SenUnitOnRoll |  |
| SenUnitCapacity |  |
| GOR (code) | H |
| GOR (name) | London |
| DistrictAdministrative (code) | E09000001 |
| DistrictAdministrative (name) | City of London |
| AdministrativeWard (code) | E05009302 |
| AdministrativeWard (name) | Cripplegate |
| ParliamentaryConstituency (code) | E14000639 |
| ParliamentaryConstituency (name) | Cities of London and Westminster |
| UrbanRural (code) | A1 |
| UrbanRural (name) | (England/Wales) Urban major conurbation |
| GSSLACode (name) | E09000001 |
| Easting | 532301 |
| Northing | 181746 |
| MSOA (name) | City of London 001 |
| LSOA (name) | City of London 001B |
| InspectorateName (name) | ISI |
| SENStat | 0 |
| SENNoStat | 22 |
| BoardingEstablishment (name) | Does not have boarders |
| PropsName | Corporation of London |
| PreviousLA (code) | 999 |
| PreviousLA (name) |  |
| PreviousEstablishmentNumber |  |
| OfstedRating (name) |  |
| RSCRegion (name) | North-West London and South-Central England |
| Country (name) |  |
| UPRN | 200000074660 |
| SiteName |  |
| QABName (code) | 0 |
| QABName (name) | Not applicable |
| EstablishmentAccredited (code) | 0 |
| EstablishmentAccredited (name) | Not applicable |
| QABReport |  |
| CHNumber |  |
| MSOA (code) | E02000001 |
| LSOA (code) | E01000002 |
| FSM | 0 |
| AccreditationExpiryDate |  |

**Reply by colmjude** [reply]
### MVP schema

There is lots of interesting data in these records.
However, keeping true to our principle of starting small we are planning to start with a schema that covers these fields

We'll call this dataset [`educational-establishments`](https://github.com/digital-land/specification/blob/main/content/dataset/educational-establishment.md?plain=1)

| field | mapped from | defaultValue |
| --- | --- | --- |
| reference | URN | |
| name | EstablishmentName | |
| educational-establishment-number | EstablishmentNumber |
| educational-establishment-status | EstablishmentStatus (code) | |
| educational-establishment-type | TypeOfEstablishment (code) | |
| open-date | OpenDate | |
| organisation | | government-organisation:D6 |
| point | Easting, Northing | | 
| local-authority-district | DistrictAdministrative (code) | | 
| school-capacity | SchoolCapacity | |
| uprn | UPRN | |
| website-url | SchoolWebsite | |
| ward | AdministrativeWard (code) | |
| entry-date | | today's date |
| start-date | LastChangedDate | |
| end-date | CloseDate | |

We've focused on capturing the location of the establishment, key information like the name and type of educational establishment, as well as links to other datasets (for example, uprns and wards) and items that might be useful for planning purposes such as school capacity.

For the data to make complete sense we'll have to create supporting datasets for `educational-establishment-status` and `educational-establishment-type`

**Reply by colmjude** [reply]
### An example record

A record on planning.data.gov.uk should look like

| field | example value |
| --- | --- |
| entity | 3100001 |
| prefix | educational-establishment |
| reference | 100001 |
| name | City of London School for Girls |
| educational-establishment-number | 6005 |
| educational-establishment-status | 1 |
| educational-establishment-type | 11 |
| open-date | 01-01-1920 |
| organisation | government-organisation:D6 |
| point | wkt... | 
| local-authority-district | E09000001 | 
| school-capacity | 860 |
| uprn | 200000074660 |
| website-url | http://www.clsg.org.uk)http://www.clsg.org.uk |
| ward | E05009302 |
| entry-date | 2024-07-10 |
| start-date | 2024-06-10 |
| end-date | |

**Reply by colmjude** [reply]
[This dataset](https://www.planning.data.gov.uk/dataset/educational-establishment) is now available on the platform.

**Comment by MrMikeRose** [comment]
### Checking we can reuse the existing dataset

The data download page states

'All content is available under the Open Government Licence v3.0, except where otherwise stated'

There is no indication that the data available for download is NOT available under this licence.

Page accessed 5/7/24 - pdf attached

[Downloads - GOV.UK.pdf](https://github.com/user-attachments/files/16109688/Downloads.-.GOV.UK.pdf)

**Comment by colmjude** [comment]
## Supporting category datasets

The educational establishment dataset has a few category or ENUM fields.

We will make the values that constrain these fields available as datasets so that the codes found in the establishment records can be mapped to the right names. For example, `EstablishmentStatus (code)` of `1` maps to `Open`.

**Reply by colmjude** [reply]
For `educational-establishment-status` we have seen the following values in the dataset of all establishments.

| reference | name | count |
| --- | --- | --- |
| 1 | Open | 27003 |
| 2 | Closed | 23847 |
| 3 | Open, but proposed to close | 112 |
| 4 | Proposed to open | 95 |

`count` = the number of records in the educational establishment dataset with this value.

**Reply by colmjude** [reply]
For `educational-establishment-type` we have seen the following value in the dataset of all the establishments

| reference | name | count |
| --- | --- | --- |
| 01 | Community school | 16432 |
| 02 | Voluntary aided school | 5136 |
| 03 | Voluntary controlled school | 3134 |
| 05 | Foundation school | 2181 |
| 06 | City technology college | 15 |
| 07 | Community special school | 1604 |
| 08 | Non-maintained special school | 94 |
| 10 | Other independent special school | 1262 |
| 11 | Other independent school | 3825 |
| 12 | Foundation special school | 128 |
| 14 | Pupil referral unit | 869 |
| 15 | Local authority nursery school | 619 |
| 18 | Further education | 489 |
| 24 | Secure units | 48 |
| 25 | Offshore schools | 154 |
| 26 | Service children's education | 83 |
| 27 | Miscellaneous | 230 |
| 28 | Academy sponsor led | 2878 |
| 29 | Higher education institutions | 138 |
| 30 | Welsh establishment | 2468 |
| 31 | Sixth form centres | 61 |
| 32 | Special post 16 institution | 150 |
| 33 | Academy special sponsor led | 101 |
| 34 | Academy converter | 7307 |
| 35 | Free schools | 551 |
| 36 | Free schools special | 111 |
| 37 | British schools overseas | 278 |
| 38 | Free schools alternative provision | 57 |
| 39 | Free schools 16 to 19 | 37 |
| 40 | University technical college | 67 |
| 41 | Studio schools | 48 |
| 42 | Academy alternative provision converter | 96 |
| 43 | Academy alternative provision sponsor led | 42 |
| 44 | Academy special converter | 319 |
| 45 | Academy 16-19 converter | 36 |
| 46 | Academy 16 to 19 sponsor led | 2 |
| 49 | Online provider | 5 |
| 56 | Institution funded by other government department | 1 |
| 57 | Academy secure 16 to 19 | 1 |

`count` = the number of records in the educational establishment dataset with this value.
